packages/react-native-bpk-component-calendar/react-native-bpk-component-calendar.podspec

```diff
@@ -0,0 +1,19 @@
+require 'json'
+
+package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
+
+Pod::Spec.new do |s|
+  s.name         = "react-native-bpk-component-calendar"
+  s.version      = package['version']
+  s.summary      = "Backpack calendar component for React Native"
+
+  s.authors      = { "backpack" => "backpacksquad@skyscanner.net" }
+  s.homepage     = "https://backpack.github.io/components/calendar"
+  s.license      = "MIT"
+  s.platform     = :ios, "9.0"
```

Comment:
Wasn't sure which platform to select, so I went with the one used in our example app podspec